docs/data/date-pickers/digital-clock/DigitalClockViews.js

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { DemoContainer, DemoItem } from '@mui/x-date-pickers/internals/demo';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { MultiSectionDigitalClock } from '@mui/x-date-pickers/MultiSectionDigitalClock';
+
+export default function DigitalClockViews() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DemoContainer
+        components={[
+          'MultiSectionDigitalClock',
+          'MultiSectionDigitalClock',
+          'MultiSectionDigitalClock',
+        ]}
+      >
+        <DemoItem label={'"hours", "minutes" and "seconds"'}>
+          <MultiSectionDigitalClock views={['hours', 'minutes', 'seconds']} />
+        </DemoItem>
+        <DemoItem label={'"hours"'}>
+          <MultiSectionDigitalClock views={['hours']} />
+        </DemoItem>
+        <DemoItem label={'"minutes" and "seconds"'}>
+          <MultiSectionDigitalClock views={['minutes', 'seconds']} />
```

Comment:
Oh, you uncovered an issue that I've left intact.
With `TimeClock` we are only rendering `ampm` toggle given an `hours` view exists. Will fix it.
In regards to `meridiem` view, it currently does not exist, because of how `TimeClock` was built and until we remove the `TimeClock` I don't see a clear solution to introducing `meridiem` view. If we do it, then the `TimeClock` would need to be adjusted/hacked to behave correctly, whereas now we are thinking about "hacking" `DigitalClock` behavior instead. 🤔